plugins/kubernetes/app/views/kubernetes/usage_limits/index.html.erb

```diff
@@ -4,8 +4,8 @@
   <%= page_title "Kubernetes Limits" %>
 
   <%= search_form do %>
-    <%= search_select :project_id, [['All', '']] + @projects.map { |p| [p.name, p.id] }, live: true, size: 3 %>
-    <%= search_select :scope_type_and_id, @env_deploy_group_array, live: true, size: 3, label: "Scope" %>
+    <%= search_select :project_id, [['All', 'all']] + @projects.map { |p| [p.name, p.id] }, live: true, size: 3 %>
```

Comment:
maybe make 'all' a constant and less likely to collide with a real id (or are id's numeric)?